services/app-web/src/pages/SubmissionReleasedToState/ReleasedToState.tsx

```diff
@@ -37,6 +37,10 @@ type ReleasedToStateValues = {
 const today = new Date()
 const ReleaseToStateSchema = Yup.object().shape({
     dateApprovalReleasedToState: Yup.date()
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+        // @ts-ignore-next-line
+        .validateDateFormat('YYYY-MM-DD', true)
```

Comment:
Sorry I missed this the first time.

**issue**: Our pattern for inline errors are to show them when you click the submit button. In this implementation we show the inline errors on input. We should change that to validate on submit. You might run into an issue where the browser input validation will prevent toggling of `shouldValidate`, you can call `preventDefault()` to stop that from happening. This same issue occurs on the rate details page.